sympy/integrals/prde.py

```diff
@@ -773,15 +773,15 @@ def limited_integrate_reduce(fa, fd, G, DE):
     c = reduce(lambda i, j: i.lcm(j), (dn,) + En)  # lcm(dn, en1, ..., enm)
     hn = c.gcd(c.diff(DE.t))
     a = hn
-    b = -derivation(hn, DE)
+    b = derivation(hn, DE)
     N = 0
 
     # These are the cases where we know that S1irr = Sirr, but there could be
     # others, and this algorithm will need to be extended to handle them.
     if DE.case in ['base', 'primitive', 'exp', 'tan']:
         hs = reduce(lambda i, j: i.lcm(j), (ds,) + Es)  # lcm(ds, es1, ..., esm)
         a = hn*hs
-        b = -derivation(hn, DE) - (hn*derivation(hs, DE)).quo(hs)
+        b = -b - (hn*b).quo(hs)
```

Comment:
this looks suspiciously wrong since it identifies derivation(hn, DE) and derivation(**hs**, DE) as b

Reply:
Definitely! Well spotted! Thanks.